docs/ums/README.md

```diff
@@ -123,16 +131,16 @@ export UMS_TOKEN=`curl -X POST -d '{
 </CodeSwitcher>
 
 ## Fetch a User
-To fetch an user you can look at the following methods:
+To fetch a user you can look at the following methods:
 
 <CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
 <template v-slot:js>
 
 ``` js
 // via alias
-const user = await ums.getUser('Elon Musk'); 
+getUser('Elon Musk').subscribe(user => console.log(user)); 
```

Comment:
Can we have an example of how to get value for observables? It was big problem for me. 
For example something like this: let curUser = ums.getUser('Elon Musk').subscribe()

Reply:
this is an example. the only way to get value from observable is to call `subscribe()` and obtain the value inside.